build-support/docker/bootstrap-docker-snapshot-image.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
The name of this script is confusing. It reads like random words strung together. This script appears to build the two images and then run an interactive bash session in a new container started from the travis image. But the name doesn't really imply any of that.

Reply:
Your description is correct, and I will edit the filename / commenting to indicate this!